tools/xskperfcompare.ps1

```diff
@@ -20,7 +20,7 @@ function ImportDataset {
     foreach ($line in $contents) {
         $array = $line.Split(",")
         $scenarioName = $array[0]
-        $scenarioData = $array[1..$array.Count]
+        $scenarioData = $array[3..$array.Count]
```

Comment:
what's this about?

Reply:
I updated the csv schema (for lack of a better word) to include two new columns, at 0-based indices 1 and 2. array[1] is now commit hash, and array[2] is unix timestamp.